samples/boards/nrf52/mesh/onoff_level_lighting_vnd_app/src/smp_svr.c

```diff
@@ -0,0 +1,79 @@
+/* Bluetooth: Mesh Generic OnOff, Generic Level, Lighting & Vendor Models
+ *
+ * Copyright (c) 2018 Vikrant More
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "common.h"
+
+#ifdef ENABLE_SMP_SVR
+
+#include <assert.h>
+#include <bluetooth/conn.h>
+#include <bluetooth/bluetooth.h>
+#include <bluetooth/gatt.h>
+#include <mgmt/buf.h>
+#include <mgmt/smp_bt.h>
+#include <stats.h>
+#include <stdlib.h>
+#include <string.h>
+#include <zephyr.h>
+
+#ifdef CONFIG_MCUMGR_CMD_IMG_MGMT
+#include "img_mgmt/img_mgmt.h"
+#endif
+#ifdef CONFIG_MCUMGR_CMD_OS_MGMT
+#include "os_mgmt/os_mgmt.h"
+#endif
+#ifdef CONFIG_MCUMGR_CMD_STAT_MGMT
+#include "stat_mgmt/stat_mgmt.h"
+#endif
+
+/* Define an example stats group; approximates seconds since boot. */
+STATS_SECT_START(smp_svr_stats)
+STATS_SECT_ENTRY(ticks)
+STATS_SECT_END;
+
+/* Assign a name to the `ticks` stat. */
+STATS_NAME_START(smp_svr_stats)
+STATS_NAME(smp_svr_stats, ticks)
+STATS_NAME_END(smp_svr_stats);
+
+/* Define an instance of the stats group. */
+STATS_SECT_DECL(smp_svr_stats) smp_svr_stats;
+
+#endif
+
+void smp_svr_init(void)
+{
+#ifdef ENABLE_SMP_SVR
```

Comment:
This file should added to the build using `zephyr_sources_ifdef` using an appropriate Kconfig option for if the smp_svr backend was enabled, rather than unconditionally building it and having it sometimes do nothing.